packages/block-editor/src/components/rich-text/style.scss

```diff
@@ -43,8 +43,14 @@
 	}
 
 	// Could be unset for individual rich text instances.
-	&.is-selected [data-rich-text-placeholder]::after {
-		display: none;
+	&.is-selected {
+		&.keep-placeholder-on-focus [data-rich-text-placeholder] {
+			pointer-events: none;
```

Comment:
Will it display a different cursor icon? I guess, it still needs to be overridden.

Reply:
It shows the same one for me.

Reply:
Why does this rule need to be added? Is the one above not enough on `:after`?

Reply:
It's needed so that you can select/see the caret through the placeholder.